content/en/api_catalog/_index.md

```diff
@@ -25,6 +25,10 @@ aliases:
 <div class="alert alert-warning">API Catalog is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
 {{< /site-region >}}
 
+<div class="alert alert-warning"><h5 class="card-title text-black mt-0 mb-1">The API Catalog is not under current development</h5>
+We are making significant improvements to the way API endpoints are represented in Datadog. The functionality currently available on this page will be moved to the <a href="https://docs.datadoghq.com/service_catalog">Service Catalog</a>. Rest assured that all core capabilities will be available in the Service Catalog after the migration is done. Stay tuned for further updates!
```

Comment:
```suggestion
We are making significant improvements to the way API endpoints are represented in Datadog. The functionality described on this page is moving to the <a href="https://docs.datadoghq.com/service_catalog">Service Catalog</a>. All core capabilities will be available in the Service Catalog after the migration is done, at which point API Catalog will be deprecated.
```

Reply:
Hi @ddOfir, I reworded to make the upcoming changes clearer, but we don't typically document future changes at all, we just update the documentation once the change is made. We do sometimes make exceptions to this when there's a good reason, though. Is there some action the user needs to take now, or some other reason they need to know about this upcoming change (such as an interruption in service)?

Reply:
Hi @ddOfir, just bumping this because there was a question there for you